timed/redmine/management/commands/update_project_expenditure.py

```diff
@@ -9,6 +9,15 @@
 class Command(BaseCommand):
     help = "Update expenditures on associated Redmine projects."
 
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--pretend",
+            dest="pretend",
+            default=True,
+            help="Pretend mode for testing",
+            type=bool,
+        )
+
```

Comment:
Add an argument as a switch with the action `store_true` or `store_false` (depending on the desired default value)

```suggestion
        parser.add_argument(
            "--pretend",
            action="store_true",
            help="Pretend mode for testing"
        )

```